tensorflow/c/kernels/BUILD

```diff
@@ -87,6 +87,22 @@ tf_cc_test(
     ],
 )
 
+tf_cc_test(
+    name = "summary_op_benchmark_test",
+    size = "small",
+    srcs = ["summary_op_benchmark_test.cc"],
+    deps = [
+        ":summary_op",
+        "//tensorflow/c:kernels",
```

Comment:
Add "//tensorflow/core:core_cpu",
Looks like it is needed for node_builder.h include.

Reply:
Looks like you added it to `summary_op_test` but it should be added to `summary_op_benchmark_test`.